corehq/apps/reports/templates/reports/reportdata/case_data.html

```diff
@@ -85,7 +85,7 @@ <h4 class="panel-title">
                               </h4>
                             </div>
                           {% endif %}
-                          <div class="panel-collapse collapse in" id="{{ table.name }}">
+                          <div class="panel-collapse collapse in" id="property-table-{{ forloop.counter }}">
```

Comment:
I believe this also needs to be applied to the other HTML ids and references that currently use `table.name`